packages/database/src/migrations/20211115202946-LESMISESSDPDrillDown5Year40District-modifies-data.js

```diff
@@ -0,0 +1,397 @@
+    },
+    {
+      groupBy: 'timestamp',
+      transform: 'mergeRows',
+    },
+  ],
+};
+
+const FRONT_END_CONFIG = {
+  name:
+    'Enrolment rate of 5-year-old children (including children with disabilities) in 40 districts (target line at 83%)',
+  type: 'chart',
+  xName: 'Year',
+  yName: 'Rate (%)',
+  chartType: 'line',
+  valueType: 'percentage',
+  chartConfig: {
+    'Priority Districts Target': {
+      color: '#4caf50',
+    },
+    'Priority Districts Average': {
+      color: '#9c27b0',
+    },
+  },
+  periodGranularity: 'year',
+};
+
+const addIndicators = async db => {
+  await insertObject(db, 'indicator', AVERAGE_INDICATOR);
+  await insertObject(db, 'indicator', TARGET_INDICATOR);
+  await insertObject(db, 'indicator', SUMMARY_INDICATOR);
+};
+
+const removeIndicators = async db => {
+  await deleteObject(db, 'indicator', { code: AVERAGE_INDICATOR.code });
+  await deleteObject(db, 'indicator', { code: TARGET_INDICATOR.code });
+  await deleteObject(db, 'indicator', { code: SUMMARY_INDICATOR.code });
+};
+
+const EXTRA_LIST_ITEMS = [
+  {
+    transform: 'updateColumns',
+    where: '=exists($er_priority_district_ece_5_summary)',
+    insert: {
+      code: 'ECEPriorityTarget5',
+      label:
+        'TARGET: Enrolment rate of 5-year-old children (including children with disabilities) in 40 districts – 2025 target of 83%',
+      parent: 'IO1_2',
+      statistic: '=$er_priority_district_ece_5_summary',
+      sort_order: '4',
+    },
+    exclude: '*',
+  },
+  {
+    transform: 'insertRows',
+    where: '=eq(@index, 1)',
+    position: 'before',
+    columns: {
+      code: 'IO1_2',
+      label:
+        'IO 1.2: Reduced gap in ECE performance between disadvantaged and non-disadvantaged areas (Part I HLO 1, IO 1.5)',
+      parent: 'HLO1',
+      statistic:
+        '=eq(max(where(f($otherRow) = equalText($otherRow.parent ? $otherRow.parent : "", "IO1_2")).statistic), min(where(f($otherRow) = equalText($otherRow.parent ? $otherRow.parent : "", "IO1_2")).statistic)) ? last(where(f($otherRow) = equalText($otherRow.parent ? $otherRow.parent : "", "IO1_2")).statistic) : 0',
+      sort_order: '3',
+    },
+  },
+];
+
+const addDrillDownToListVisual = async db => {
+  // Add drillDown to dashboard_item config
+  const dashboardItem = await findSingleRecord(db, 'dashboard_item', {
+    code: 'LESMIS_ESSDP_ECE_HLO1',
+  });
+  dashboardItem.config.drillDown.itemCodeByEntry = {
+    ...dashboardItem.config.drillDown.itemCodeByEntry,
+    ECEPriorityTarget5: DRILL_DOWN_CODE,
+  };
+  await updateValues(
+    db,
+    'dashboard_item',
+    { config: dashboardItem.config },
+    { code: 'LESMIS_ESSDP_ECE_HLO1' },
+  );
+  // Add summary to list report
+  const listReport = await findSingleRecord(db, 'report', {
+    code: 'LESMIS_ESSDP_ECE_HLO1',
+  });
+  listReport.config.fetch.dataElements.push('er_priority_district_ece_5_summary');
+  listReport.config.transform.splice(1, 0, ...EXTRA_LIST_ITEMS);
```

Comment:
Splice the new list entries in after the the `keyValueByDataElementName`

Since we build lists in reverse, and we know these new entries need to appear at the bottom of the list, we know we can insert them at the top